gradle.properties

```diff
@@ -11,7 +11,7 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-VERSION_NAME=1.1.0
```

Comment:
bump version to `2.0.0` pls

Reply:
85d46e6411dd668cfeb6380ee98920b2b7d8b622